pkg/controller/cloud/node_controller.go

```diff
@@ -219,7 +219,7 @@ func (cnc *CloudNodeController) AddCloudNode(obj interface{}) {
 
 	cloudTaint := getCloudTaint(node.Spec.Taints)
 	if cloudTaint == nil {
-		klog.V(2).Infof("This node %s is registered without the cloud taint. Will not process.", node.Name)
+		klog.V(5).Infof("This node %s is registered without the cloud taint. Will not process.", node.Name)
```

Comment:
This only applies to `Add` events which should _ideally_ only happen once when the node is registered. Do you think it's still worth increasing the log level here?

Reply:
well at least for me this is spamming all the time. And nodes are registered (zone and region labels are there)

Reply:
The PR to update this went in recently https://github.com/kubernetes/kubernetes/pull/65981 so you are likely running a version that does not have this update yet.

Reply:
Previously it would log for both Add and Update events.

Reply:
yep I am using 1.11.6 where I tested it

Reply:
okay, so getting this merged into master probably doesn't make sense but we can consider back porting it to previous versions if the log messages are too noisy